spec/models/reservation_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Reservation, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] This file has no implemented examples.  Kindly remove it if not using it. 👍